vendor/bat-native-ledger/src/bat/ledger/internal/properties/batch_proof_properties.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/properties/batch_proof_properties.h"
+
+namespace ledger {
+
+BatchProofProperties::BatchProofProperties() = default;
+
+BatchProofProperties::BatchProofProperties(
+    const BatchProofProperties& properties) {
+  transaction = properties.transaction;
+  ballot = properties.ballot;
+}
```

Comment:
same as above

Reply:
Complex class/struct needs an explicit out-of-line copy constructor as per Chromium style guide